src/components/country-challenges-chart/country-challenges-chart-selectors.js

```diff
@@ -74,10 +118,12 @@ const getYAxisTicks = createSelector(
 
 
 const mapStateToProps = createStructuredSelector({
-  data: getFilteredData,
+  data: getFilteredAvailableData,
```

Comment:
```suggestion
  data: getFilteredData,
```

Reply:
We need the getFilteredAvailableData to be able to remove the countries bubbles when a specific country is selected and they don`t have value for the indicator selected. See the second case on the task description and testing instructions